src/AppInstallerCLICore/Argument.h

```diff
@@ -47,24 +47,24 @@ namespace AppInstaller::CLI
         };
 
         // Defines an argument with no alias.
-        constexpr static char NoAlias = '\0';
+        constexpr static char const* NoAlias = "\0";

```

Comment:
```suggestion
        constexpr static std::string_view NoAlias;
```
Can just be empty string.